doc/configuration.md

```diff
@@ -29,13 +29,13 @@ Parameters controlling the Cluster-API management server (capi management server
 
 Parameters controlling both management server creation and cluster creation:
 
-| environment         | clusterctl.yaml                 | provenance | default                              | meaning                                                                                                                      |
-|---------------------|---------------------------------|------------|--------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
-| `cloud_provider`    | `OPENSTACK_CLOUD`               | capo       |                                      | `OS_CLOUD` name in clouds.yaml                                                                                               |
-| `external`          | `OPENSTACK_EXTERNAL_NETWORK_ID` | capo       | `""`                                 | Name/ID of the external (public) OpenStack network, default "" uses the detected external network                            |
-| `dns_nameservers`   | `OPENSTACK_DNS_NAMESERVERS`     | capo       | `[ "5.1.66.255", "185.150.99.255" ]` | Array of nameservers for capi mgmt server and for cluster nodes, replace the FF MUC defaults with local servers if available |
-| `availability_zone` | `OPENSTACK_FAILURE_DOMAIN`      | capo       |                                      | Availability Zone(s) for the mgmt node / workload clusters                                                                   |
-| `kind_mtu`          | `MTU_VALUE`                     | SCS        | `0`                                  | MTU for the mgmt server; Calico is set 50 bytes smaller; 0 means autodetection                                               |
+| environment         | clusterctl.yaml                 | provenance | default                              | meaning                                                                                                                                                                                           |
+|---------------------|---------------------------------|------------|--------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `cloud_provider`    | `OPENSTACK_CLOUD`               | capo       |                                      | `OS_CLOUD` name in clouds.yaml                                                                                                                                                                    |
+| `external`          | `OPENSTACK_EXTERNAL_NETWORK_ID` | capo       | `""`                                 | Name of the external (public) OpenStack network, default "" uses the detected external network(for clouds with one external network); use `external_id` if you want to specify network ID instead |
```

Comment:
This is a bit confusing, because in the env file, this accepts only name, but in the `clusterctl.yaml` I would expect given the name an ID there, and what I find in the default one generated is indeed ID, but in the docs, it says a name should be there.

Maybe something like this?

```suggestion
| `external`/`external_id`          | `OPENSTACK_EXTERNAL_NETWORK_ID` | capo       | `""`                                 | Name or ID of the external (public) OpenStack network, default "" uses the detected external network(for clouds with one external network); use `external_id` if you want to specify network ID in the env file instead |
```

Reply:
Please see the newest changes, I simplified it a little bit.